e2e_testing/xfail_sets.py

```diff
@@ -12,7 +12,9 @@
 
 from torch_mlir_e2e_test.test_suite import COMMON_TORCH_MLIR_LOWERING_XFAILS
 
-LINALG_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS
+LINALG_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS.union({
+    "Quantize_uint8"
```

Comment:
In your description you suggest you want help understanding why this fails. I suggest you also add that in a comment here.